redux-form/lib/Field.d.ts

```diff
@@ -72,15 +66,22 @@ interface BaseFieldProps {
      * should return `undefined`, if the field is invalid, it should return an error
      * (usually, but not necessarily, a `String`).
      */
-    validate?: Validator|Validator[];
+    validate?: Validator;
 
     /**
      * Allows you to to provide a field-level warning rule. The function will be given the
      * current value of the field and all the other form values. If the field needs a warning,
      * it should return the warning (usually, but not necessarily, a `String`). If the field
      * does not need a warning, it should return `undefined`.
      */
-    warn?: Validator|Validator[];
+    warn?: Validator;
```

Comment:
Both `validate` and `warn` can be a list of Validators
http://redux-form.com/6.5.0/examples/fieldLevelValidation/